Plugins/SequencePlugin/Source/SequencePlugin/Public/Marketplace/Structs/SeqMarketplacePage.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "SeqMarketplaceSortBy.h"  // Ensure this includes your SortBy struct or enum
+#include "SeqMarketplacePage.generated.h"
+
+USTRUCT(BlueprintType)
+struct SEQUENCEPLUGIN_API FSeqMarketplacePage
```

Comment:
Do we need another Page definition or can we use the Page defined for the Indexer?

Reply:
Hey @BellringerQuinn, can I ask which other Page Definition you're referring to?

Reply:
the indexer struct uses page as well, however as i didnt want to include the indexer structs on the marketplace object, i made another one.